### ISSUE TITLE
More padding for column checkboxes

### ISSUE THREAD

**Issue by brynwhyman** [silverstripe/silverstripe-ckan-registry#101]
Update styling so the whole cell/padding for column's 'in results'/'in detail' checkbox is selectable

**Comment by robbieaverill** [comment]
Thoughts @brynwhyman?

**Comment by ScopeyNZ** [comment]
I agree it belongs on GridFieldExtensions. I think there should be a variant (probably through configuration) that allows label-less checkboxes in columns. It can still be screen reader friendly with appropriate html attributes. I personally would prefer that it also involve making the whole table cell the label for the checkbox - ie. making a click anywhere on the cell toggle the checkbox.

**Comment by brynwhyman** [comment]
Moved to: https://github.com/symbiote/silverstripe-gridfieldextensions/issues/281

I'll keep this ticket in CKAN to track whether we action this as part of the CKAN initiative currently underway.
